### DISCUSSION TITLE
Strategy to reference a net core projecct

### DISCUSSION THREAD

**Discussion by juanan** [dotnet/aspnetcore#28142] [General]
Hi,

I know that the recommended way to add a reference to a net core library is via Nuget but since this is a really inconvenient in some scenarios, is there other way?

I can add a direct reference to the output dll and the IDE works ok but when I publish I miss all the referenced library dependencies so it should exist another way (I don't find any place talking about alternatives).

Thanks a lot.

**Comment by davidfowl** [comment]
> I can add a direct reference to the output dll and the IDE works ok but when I publish I miss all the referenced library dependencies so it should exist another way (I don't find any place talking about alternatives).

This is just a local reference on hard drive somewhere? This is a question for the SDK https://github.com/dotnet/sdk

**Reply by juanan** [reply]
Yes, its a local reference.

I know that via Nuget I would get all the dependencies but I am wondering if I can use other approach.
